dll/appcompat/apphelp/shimeng.c

```diff
@@ -207,6 +208,35 @@ BOOL WINAPIV SeiDbgPrint(SEI_LOG_LEVEL Level, PCSTR Function, PCSTR Format, ...)
     return TRUE;
 }
 
+static
+BOOL SeiIsOrdinalName(LPCSTR lpProcName)
+{
+    return (ULONG_PTR)lpProcName <= MAXUSHORT;
```

Comment:
Maybe worth converting this to a macro that does: `(((ULONG_PTR)(lpProcName) >> 16) == 0)`
(similar to IS_INTRESOURCE)

Reply:
Why?

Reply:
🤔